docs/source/usage/workflows/ionization.rst

```diff
@@ -0,0 +1,187 @@
+            using ParticleFlagsElectrons = MakeSeq_t<
+                particlePusher<UsedParticlePusher>,
+                shape<UsedParticleShape>,
+                interpolation<UsedField2Particle>,
+                current<UsedParticleCurrentSolver>,
+                massRatio<MassRatioElectrons>,
+                chargeRatio<ChargeRatioElectrons>>;
+
+            using ParticleAttributesElectrons = MakeSeq_t<position<position_pic>, momentum, weighting>;
+
+            using Electrons = Particles<PMACC_CSTRING("e"), ParticleFlagsElectrons, ParticleAttributesElectrons>;
+        } // namespace picongpu
+
+
+5. Define for each ion species a PIConGPU species
+
+    The definition of an ioniaztion species must include the particle attribute :code:`boundElectrons`, and the particle flags :code:`atomicNumbers`, :code:`ionizationEnergies`, :code:`effectiveNuclearCharge`, :code:`ionizers`, :code:`massRatio` and :code:`chargeRatio`.
+
+    .. code-block:: c++
+
+        namespace picongpu
+        {
+            #ifndef PARAM_IONIZATIONCURRENT
+            #    define PARAM_IONIZATIONCURRENT None
+            #endif
+
+            /* Example Nitrogen */
+            using ParticleFlagsNitrogen = MakeSeq_t<
+                particlePusher<UsedParticlePusher>,
+                shape<UsedParticleShape>,
+                interpolation<UsedField2Particle>,
+                current<UsedParticleCurrentSolver>,
+                atomicNumbers<ionization::atomicNumbers::Nitrogen>                                                 // <-- from step 1
+                effectiveNuclearCharge<ionization::effectiveNuclearCharge::Nitrogen_t>, // <-- from step 1
+                ionizationEnergies<ionization::energies::AU::Nitrogen_t >               // <-- from step 2
+                massRatio<MassRatioNitrogen>,                                           // <-- from step 3
+                chargeRatio<ChargeRatioNitrogen>,                                       // <-- from step 3
+                ionizers<MakeSeq_t<               // <-- comma separated list of all ionizers of this species with
+                                                  // species of electrons macro particles to be created upon ionization
+                    particles::ionization::BSIEffectiveZ<Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT>,
+                    particles::ionization::ADKLinPol<Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT>,
+                    particles::ionization::ThomasFermi<Electrons>>>>;
+
+            using ParticleAttributesNitrogen = MakeSeq_t<position<position_pic>, momentum, weighting, boundElectrons>;
+            using Nitrogen = Particles<PMACC_CSTRING("N"), ParticleFlagsNitrogen, ParticleAttributesNitrogen>;
+        } // namespace picongpu
+
+    .. note::
+        Remember that you can define multiple electron species and associate them with your different ion species to track the ionisation processes separately!
```

Comment:
change from AE to BE - please stick to AE
```suggestion
        Remember that you can define multiple electron species and associate them with your different ion species to track the ionization processes separately!
```

Reply:
done